client/src/components/PlaylistUser/PlaylistUser.js

```diff
@@ -1,9 +1,35 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { authObserverLoading } from "../../redux/auth/action";
+import { getFavoritePlaylists } from "../../redux/playlist/action";
+import { useHistory } from "react-router";
+import { useLocation } from "react-router-dom";
 import "./style/playlistuser.scss";
 import backgroundPicture from "../../assets/images/background/profile-picture-background.jpg";
 import editIcon from "../../assets/images/icons/editIcon.png";
 function PlaylistUser({ playlistUserData }) {
-  console.log(playlistUserData);
+  const history = useHistory();
+  const location = useLocation();
+  const dispatch = useDispatch();
+  const { user, loading, authObserverSuccess, signOutSuccess } = useSelector(
+    (state) => state.auth
+  );
+  const { myFavoritePlaylists, playlistCreatedSuccess } = useSelector(
+    (state) => state.playlist
+  );
+
+  useEffect(() => {
+    if (!loading && authObserverSuccess) {
+      dispatch(getFavoritePlaylists(playlistUserData.firebase_id));
+    }
+  }, [loading, authObserverSuccess, playlistCreatedSuccess]);
+  useEffect(() => {
+    dispatch(authObserverLoading());
+    if (signOutSuccess) {
+      history.push("/login");
+    }
+  }, []);
```

Comment:
We are not using this anymore on the components, it's already on the nav bar. Will remove it after merging :)